README.md

```diff
@@ -36,6 +36,27 @@ dependencies {
     testImplementation("app.futured.arkitekt:cr-usecases-test:LatestVersion")
 }    
 ```
+
+## Snapshot installation
+Create access token in your GitHub Account settings
+**Settings** -> **Developer settings** -> **Personal access tokens** -> **Generate new token**
+
+Check **read:packages**
+
+add new maven repo to your top level gradle file
+
+```
+maven { 
+    url = URI("https://maven.pkg.github.com/futuredapp/arkitekt")
+    credentials {
```

Comment:
Nedá sa to nejako použiť bez toho aby sme museli zadávať credentials? Napadá ťa prečo to je spravené takýmto spôsobom?

Reply:
Řekl bych že je to tím, že GH packages má limitovanej trafic na 10GB měsíčně, takže nechtějí aby se z toho stala alternativa pro jitpack nebo jcenter